Extensions/retags.js

```diff
@@ -119,16 +121,17 @@ XKit.extensions.retags = {
 				$target = $element.find('.notification-wrapper');
 				url = $target.find('a').not('.notification-username').attr('href');
 			}
-			if (url) {
+			if (url && !host && !id) {
 				url = url.split('/');
-				var host = url[2], id = url[4];
-				XKit.extensions.retags.request(host, id).then(function(tags) {
-					XKit.extensions.retags.append_tag($element, retagClass, $target, tags);
-				}).fail(function(errorResponse) {
-					var tagError = 'ERROR: ' + errorResponse.status;
-					XKit.extensions.retags.append_tag($element, retagClass, $target, tagError);
-				});
+				host = url[2];
+				id = url[4];
 			}
+			XKit.extensions.retags.request(host, id).then(function(tags) {
```

Comment:
this should be in a guard expression like if(host), to preserve the behavior before this update